cfme/tests/cli/test_evmserverd.py

```diff
@@ -42,10 +42,19 @@ def test_evmserverd_stop(appliance, request):
 
     server_name_key = 'Server'
 
+    @wait_for_decorator(timeout="2m", delay=5)
+    def get_server_names():
+        for server in appliance.ssh_client.status["servers"]:
+            if server.get(server_name_key):
```

Comment:
This can likely be flattened a bit to: `return bool(server.get(server_name_key, False))`

Please include a docblock here of what this is waiting for, the behavior we're trying to mitigate. This will stop waiting once a single element in the status has `server_name_key`, why doesn't it wait for _all_ of them?

Reply:
Good point, thanks for pointing out my faulty logic :smile: !!